### DISCUSSION TITLE
v1.97.1

### DISCUSSION THREAD

**Discussion by cyring** [cyring/CoreFreq#480] [Show and tell]
**Full Changelog**: https://github.com/cyring/CoreFreq/compare/1.96.5...1.97.1

<hr /><em>This discussion was created from the release <a href='https://github.com/cyring/CoreFreq/releases/tag/1.97.1'>v1.97.1</a>.</em>

**Comment by cyring** [comment]
@bubbleguuum @inglor @ich777

Hello,

For packaging purposes, version 1.97.1 is available.
See above, the `241` commit comments are making out the Changelog.

![2024-04-05-140659_644x1012_scrot](https://github.com/cyring/CoreFreq/assets/11563789/c2238b2e-170c-409e-939c-1ee681054e61)

Best regards
Cyril

**Comment by bubbleguuum** [comment]
Congrats for the release!

Package updated:
https://build.opensuse.org/package/show/hardware/CoreFreq

**Reply by cyring** [reply]
Thank you very much.
